packages/web-components/src/components/ic-top-navigation/ic-top-navigation.stories.mdx

```diff
@@ -590,96 +284,60 @@ import NavigationGroup from "../ic-navigation-group/readme.md";
-              viewBox="0 0 24 24"
-              width="24px"
-              fill="#000000"
-            >
+          <ic-search-bar slot="search" placeholder="Search" label="Search"></ic-search-bar>
+          <ic-navigation-button label="button1" slot="buttons" onclick="alert('test')">
+            <svg slot="icon" xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000">
               <path d="M0 0h24v24H0V0z" fill="none" />
               <path
                 d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm-5.5-2.5l7.51-3.49L17.5 6.5 9.99 9.99 6.5 17.5zm5.5-6.6c.61 0 1.1.49 1.1 1.1s-.49 1.1-1.1 1.1-1.1-.49-1.1-1.1.49-1.1 1.1-1.1z"
               />
             </svg>
           </ic-navigation-button>
-          <ic-navigation-item
-            slot="navigation"
-            label="Navigation"
-            href="/"
-            selected="true"
-          ></ic-navigation-item>
-          <ic-navigation-item
-            slot="navigation"
-            label="Navigation"
-            href="/"
-          ></ic-navigation-item>
-          <ic-navigation-item
-            slot="navigation"
-            label="Navigation"
-            href="/"
-          ></ic-navigation-item>
-          <ic-navigation-item
-            slot="navigation"
-            label="Navigation"
-            href="/"
-          ></ic-navigation-item>
-          <ic-navigation-item
-            slot="navigation"
-            label="Navigation"
-            href="/"
-          ></ic-navigation-item>
+          <ic-navigation-item slot="navigation" label="Navigation" href="/" selected="true"></ic-navigation-item>
+          <ic-navigation-item slot="navigation" label="Navigation" href="/"></ic-navigation-item>
+          <ic-navigation-item slot="navigation" label="Navigation" href="/"></ic-navigation-item>
+          <ic-navigation-item slot="navigation" label="Navigation" href="/"></ic-navigation-item>
+          <ic-navigation-item slot="navigation" label="Navigation" href="/"></ic-navigation-item>
         </ic-top-navigation>
         <br />
-        <ic-button
-          variant="primary"
-          id="default-btn"
-          onClick="document.querySelector('ic-theme').color='rgb(27, 60, 121)'"
-        >
-          Default theme
-        </ic-button>
-        <ic-button
-          variant="primary"
-          id="custom-btn"
-          onClick="document.querySelector('ic-theme').color='rgb(255, 201, 60)'"
-        >
-          Sunrise theme
-        </ic-button> `
+        <ic-button variant="primary" id="default-btn" onClick="document.querySelector('ic-theme').color='rgb(27, 60, 121)'"> Default theme </ic-button>
+        <ic-button variant="primary" id="custom-btn" onClick="document.querySelector('ic-theme').color='rgb(255, 201, 60)'"> Sunrise theme </ic-button> `
+    }
+  </Story>
+</Canvas>
+
+### Long single-word app title
```

Comment:
I think we want this same story adding to the React storybook as well (react/src/stories)